lib/galaxy/tools/xsd/galaxy.xsd

```diff
@@ -1990,6 +1990,33 @@ multiple formats.
        help="The set of aligned reads." />
 ```
 
+Perhaps counter-intuitively, a ``multiple="true"`` data parameter requires at least one
+data input. If ``optional="true"`` is specified, this condition is relaxed and the user
+is allowed to select 0 datasets. Unfortunately, if 0 datasets are selected the resulting
+value for the parameter during Cheetah templating (such as in a ``command`` block) will
+effectively a list with one ``None``-like entity in it.
+
+The following idiom can be used to iterate over such a list and build a hypothetical ``-B``
+parameter for each file - the ``if`` block is used to handle the case where a ``None``-like
+entity appears in the list because no files were selected: 
+
+```
+#for $input in $input1
+#if $input
+-B "$input"
+#end if
+#end for
+```
```

Comment:
The test tool is failing, not sure this syntax is the problem, but I know of another way to process `$input1`:
```
#if str($input1) != 'None'
    #for $input in $input1
        -B "$input"
    #end for
#end if
```
See e.g. https://github.com/galaxyproject/tools-iuc/blob/master/tools/abyss/abyss-pe.xml#L111
I think that ```#if len($input1) != 0``` may be also used as initial test, see https://github.com/galaxyproject/tools-iuc/blob/master/tools/tool_factory_2/rgToolFactory2.xml#L47